source/hardware/controllers/hercules_djcontrol_MIX.rst

```diff
@@ -0,0 +1,116 @@
+Hercules DJControl MIX
+==============================
+
+-  `Manufacturer’s product page <https://www.hercules.com/en-us/product/djcontrol-mix/>`__
+-  `Manufacturer’s support and downloads page <https://support.hercules.com/en/product/djcontrolmix-en/>`__
+-  `Forum thread <https://mixxx.discourse.group/t/hercules-contrl-mix-mapping/26581>`__
+
+.. versionadded:: 2.3
```

Comment:
So I modified the file with new versionadded:: 2.4.0 in a new pull request. Was that correct ?

Reply:
Just enter 2.4.0 here, no new PR needed.

Reply:
I wish I knew how to do this

Reply:
Just change line 8 to `.. versionadded:: 2.4.0`

Reply:
And then stage your changes (`git add -p`), commit them (`git commit`) and push the new commit to this branch (`git push`).

Reply:
> Just change line 8 to...

Which is what I did, and somehow created a new pull request (https://github.com/mixxxdj/manual/pull/582)

My comment should have been : I wish I knew how to do this properly through this web interface (if that is ever possible)...

> And then stage your changes...

And same applies here. Do we need to rely on command lines or this can be done from here ?

Reply:
Just edit the text in line 8 of the file, as suggested above. Not local Git client needed for this.